jwql/website/apps/jwql/templates/archive.html

```diff
@@ -2,73 +2,118 @@
 
 {% block preamble %}
 
-	<title>Archived {{ inst }} Images - JWQL</title>
+    <!-- Custom styles and scripts for this template -->
+    <style type="text/css">
+        div#proposal-array { display: none; }
+    </style>
+
+    <script type="text/javascript">
+    /**
+     * Updates various compnents on the archive page
+     * @param {String} inst - The instrument of interest (e.g. "FGS")
+     * @param {String} base_url - The base URL for gathering data from the AJAX view.
+     */
+    function update_archive_page(inst, base_url) {
+        $.ajax({
+            url: base_url + '/ajax/' + inst + '/archive/',
+            success: function(data){
+
+                // Update the number of proposals displayed
+                num_proposals = data.thumbnails.proposals.length;
+                update_show_count(num_proposals, 'proposals')
+
+                // Add content to the proposal array div
+                for (var i = 0; i < data.thumbnails.proposals.length; i++) {
+
+                    // Parse out useful variables
+                    prop = data.thumbnails.proposals[i];
+                    thumb = data.thumbnails.thumbnail_paths[i];
+                    n = data.thumbnails.num_files[i];
+
+                    // Build div content
+                    content = '<div class="proposal text-center">';
+                    content += '<a href="/' + inst + '/archive/' + prop + '/" id="proposal' + i + '" proposal="' + prop + '"';
```

Comment:
Okay, I figured out the search issue. Weird that still don't see it...

The javascript function that decides which thumbnails to hide and show when a value is entered into the search bar looks at the different id values for the thumbnails. These are `id=proposal#` where the `#` was originally taken from the loop iterator in Django, and started at 1. The new way (here) is a javascript for loop that starts at 0! 

When I tested it, changing the `i` in this line to `(i + 1)` fixed the problem.